### PR TITLE
Feature/basic main

### PR DESCRIPTION
**Pull request by nils-schmitt**
The basic implementation of the pipeline is now complete. This PR contains a solution for issues #8 and #10 .

### CODE REVIEW

File: content/outputs/intermediates/bulletpoints_withEnd.txt
Comment (PitButtchereit) [inline content/outputs/intermediates/bulletpoints_withEnd.txt:1]: In this file bulletpoints have a start and end date. Therefore it should be renamed to match the content.

"bulletpoints_with_start_end.txt"

**Review by PitButtchereit — COMMENTED** [review]
Looks like a good starting point for the project. One file should be renamed.

Styling does not follow the PEP 8 standard yet. These workflows will be added to the repository shortly.

File: content/outputs/intermediates/bulletpoints.txt
Comment (FR-SON) [inline content/outputs/intermediates/bulletpoints.txt:1]: All files in "intermediates" are creates in the process of on specific extraction. Should they be part of the repo? I think they don't need to be pushed. We should think about that again later. For now, it's fine.

**Review by FR-SON — APPROVED** [review]
Very nice base for our future developments. As already mentioned, styling according to newly determined guidelines will be implemented during the next refactoring iteration.

**Review by PitButtchereit — APPROVED** [review]
Looks good

**Pull request merged**